build-tools/packages/build-cli/docs/update-dep.md

```diff
@@ -0,0 +1,34 @@
+`flub update-dep`
```

Comment:
Suggest either making this `modify dependency` or maybe `modify override`. Regardless, make it a verb:noun combination.

Reply:
Updated to `modify lockfile`.